workspace.json

```diff
@@ -3066,6 +3094,172 @@
+              "apps/services/auth-admin-api/tsconfig.app.json",
+              "apps/services/auth-admin-api/tsconfig.spec.json"
+            ],
+            "exclude": [
+              "**/node_modules/**",
+              "!apps/services/auth-admin-api/**"
+            ]
+          }
+        },
+        "test": {
+          "builder": "@nrwl/jest:jest",
+          "options": {
+            "jestConfig": "apps/services/auth-admin-api/jest.config.js",
+            "tsConfig": "apps/services/auth-admin-api/tsconfig.spec.json",
+            "passWithNoTests": true
+          }
+        }
+      }
+    },
+    "services-auth-api": {
+      "root": "apps/services/auth-api",
+      "sourceRoot": "apps/services/auth-api/src",
+      "projectType": "application",
+      "prefix": "auth-api",
+      "schematics": {},
+      "architect": {
+        "build": {
+          "builder": "@nrwl/node:build",
+          "options": {
+            "outputPath": "dist/apps/services/auth-api",
+            "main": "apps/services/auth-api/src/main.ts",
+            "tsConfig": "apps/services/auth-api/tsconfig.app.json",
+            "assets": ["apps/services/auth-api/src/assets"]
+          },
+          "configurations": {
+            "production": {
+              "optimization": true,
+              "extractLicenses": true,
+              "inspect": false,
+              "fileReplacements": [
+                {
+                  "replace": "apps/services/auth-api/src/environments/environment.ts",
+                  "with": "apps/services/auth-api/src/environments/environment.prod.ts"
+                }
+              ]
+            }
+          }
+        },
+        "serve": {
+          "builder": "@nrwl/node:execute",
+          "options": {
+            "buildTarget": "services-auth-api:build"
+          }
+        },
+        "lint": {
+          "builder": "@nrwl/linter:lint",
+          "options": {
+            "linter": "eslint",
+            "config": "apps/services/auth-api/.eslintrc",
+            "tsConfig": [
+              "apps/services/auth-api/tsconfig.app.json",
+              "apps/services/auth-api/tsconfig.spec.json"
+            ],
+            "exclude": ["**/node_modules/**", "!apps/services/auth-api/**"]
+          }
+        },
+        "test": {
+          "builder": "@nrwl/jest:jest",
+          "options": {
+            "jestConfig": "apps/services/auth-api/jest.config.js",
+            "tsConfig": "apps/services/auth-api/tsconfig.spec.json",
+            "passWithNoTests": true
+          }
+        },
+        "docker-express": {},
+        "dev-services": {
+          "builder": "@nrwl/workspace:run-commands",
+          "options": {
+            "commands": [
+              {
+                "command": "docker-compose -f docker-compose.base.yml -f docker-compose.dev.yml up -d"
+              }
+            ],
+            "cwd": "apps/services/auth-api"
+          }
+        },
+        "generate-schema": {
```

Comment:
We might want to rename this to `init-schema` so we can gitignore the schema and generate it in `postinstall` and in `CI`.

I see that this is currently pretty random and confusing, looking across workspace.json, but we plan to simplify it a bunch (see 
#1225).